apps/aeutils/priv/extensions/export_chain.sh

```diff
@@ -1,59 +1,11 @@
-#!/bin/bash
```

Comment:
I did not find automated tests for this script. Can you please confirm that it still works.

Reply:
It works. Tested on Windows and Unix.

Reply:
Thanks.